app/jobs/resolution_proofing_job.rb

```diff
@@ -19,8 +19,7 @@ def perform(
     encrypted_arguments:,
     trace_id:,
     should_proof_state_id:,
-    double_address_verification: false,
-    ipp_enrollment_in_progress: false,
+    ipp_enrollment_in_progress:,
```

Comment:
Will we need to split this over several deploys to gracefully handle 50/50 states?

Ref: https://handbook.login.gov/articles/manage-50-50-state.html#jobs

Reply:
It looks like this pull request includes both of Deploy 2 ("Remove the argument in calls to the job") and Deploy 3 ("Remove the argument") from under "Remove an argument from a job #perform method".

Remove the argument in calls to the job: https://github.com/18F/identity-idp/pull/9829/files#diff-b4c1fa063fe7add8eae466de029891f060102bf5c16e4e69e88fe76e5676332fL31

Remove the argument [in the job itself]: https://github.com/18F/identity-idp/pull/9829/files#diff-32ee3d7bfb2561fc0fdef096c00c6533d565fb019e842a72be42b16b9fdaa959L22

Reply:
Yeah thanks @aduth this PR should be broken into two. @svalexander I didn't quite get it right with the first PR you referenced. That one made sure the jobs are aware of `ipp_enrollment_in_progress` and that the agent.rb file passes in the correct value to the jobs.